extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -120,7 +121,8 @@ export class RefreshManager {
     this.adType_ = this.element_.getAttribute('type').toLowerCase();
 
     /** @const @private {?number} */
-    this.refreshInterval_ = this.getPublisherSpecifiedRefreshInterval_();
+    this.refreshInterval_ = (refreshInterval * 1000) ||
```

Comment:
I'd suggest that it would be better to call this refreshIntervalMsecs_.

Reply:
+1

Reply:
Done.